src/algorithm/dynamics.hpp

```diff
@@ -37,6 +37,7 @@ namespace se3
   ///           \text{s.t.} & & J (q) \ddot{q} + \gamma (q, \dot{q}) = 0 \end{eqnarray} \f$ </CENTER> <BR>
   ///       where \f$ \ddot{q}_{\text{free}} \f$ is the free acceleration (i.e. without constraints),
   ///       \f$ M \f$ is the mass matrix, \f$ J \f$ the constraint Jacobian and \f$ \gamma \f$ is the constraint drift.
+  ///  By default, the constraint Jacobian is assumed to be full rank, and undamped cholesky inverse is performed.
```

Comment:
Cholesky and not chloesky ;)

Reply:
I assume you meant cholesky, not chloesky :trollface: :stuck_out_tongue_winking_eye: